src/main/java/seedu/address/model/question/MultipleChoiceQuestion.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address.model.question;
+
+import java.util.Set;
+
+import seedu.address.model.choice.Choice;
+import seedu.address.model.tag.Tag;
+
+public class MultipleChoiceQuestion extends Question {
+    public MultipleChoiceQuestion(Name name, Phone phone, Email email, Address address, Set<Tag> tags,
```

Comment:
Maybe you could remove the `phone`, `email`, `address`?

Reply:
I think this will require a lot more changes and changing a lot of the tests. This PR already contains many changes to files so I think it should be merged before we attempt to remove the fields. This can be done in a separate issue and PR so we don't have too many changes in one PR